### PR TITLE
addition, subtraction, multiplication, division works

### PR DESCRIPTION
**Pull request by aniolm**
First part of cpp-fundamentals homework.

### CODE REVIEW

**Comment by ziobron** [comment]
⏱ 3 XP extra for @aniolm for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -1,7 +1,31 @@
 #pragma once
 #include <string>
+#include <map>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+
+std::map<std::string,int> commandMap{{"add", 1},{"subtract", 2},{"multiply",3},{"divide",4}};
+int commandInt = commandMap[command];
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:8]: Unnecessary. You complicated code here. Better use if.

File: homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -1,7 +1,31 @@
...
+                        return std::to_string(first+second);
+                        break;
+                case 2:
+                        return std::to_string(first-second);
+                        break;
+                case 3:
+                        return std::to_string(first*second);
+                        break;
+                case 4:
+                       if(second>0){
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:21]: Add space between operations like '+' '-' '!=' '==' etc...

File: homework/calculate/calculate.hpp [hunk 3]
```diff
@@ -1,7 +1,31 @@
...
+                        return std::to_string(first/second);
+                       }else{
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:23]: Add space between `else` `for` etc..

File: homework/calculate/main.cpp [hunk 4]
```diff
@@ -1,6 +1,6 @@
 #include <iostream>
 #include <string>
-#include "calculate.hpp"
+#include "calculate.hpp"
```
Comment (nauka-programowania-MA) [inline homework/calculate/main.cpp:3]: Do not add white spaces at the end of line

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**